randovania/games/prime1/exporter/game_exporter.py

```diff
@@ -7,6 +7,8 @@
 
 import py_randomprime
 
+import PARAMETEREDITOR.PARAMETEREDITOR
```

Comment:
That's a really weird name module name you used for the package.

Reply:
ngl if I could go back and tell myself to change it to something better I would....idk what I was thinking.